src/rgw/rgw_sal.h

```diff
@@ -85,6 +73,10 @@ class RGWStore : public DoutPrefixProvider {
     virtual bool is_meta_master() = 0;
     virtual int forward_request_to_master(RGWUser* user, obj_version *objv,
 				  bufferlist& in_data, JSONParser *jp, req_info& info) = 0;
+    virtual int defer_gc(RGWObjectCtx *rctx, RGWBucket* bucket, RGWObject* obj,
+			 optional_yield y) = 0;
+    virtual const RGWZoneGroup& get_zonegroup() = 0;
+    virtual int get_zonegroup(const string& id, RGWZoneGroup& zonegroup) = 0;
```

Comment:
i'm not sure how much multisite configuration stuff we're gonna want under the store interface - especially if there's a future where the multisite configuration itself describes what kind of stores get constructed and used. maybe that deserves a separate interface outside of Store? something to think about

Reply:
Yeah, that may be.  I'll start thinking about what that would look like.  In the meantime, I'll move these APIs into zipper to help get rid of the ->getRados() calls.